R/expand.R

```diff
@@ -277,7 +277,7 @@ grid_dots <- function(..., `_data` = NULL) {
     }
 
     arg <- paste0("..", i)
-    vec_assert(dot, arg = arg)
+    vec_assert(dot, arg = arg, call = caller_env())
```

Comment:
Probably worth adding `error_call = caller_env()` as an argument to `grid_dots()`?